src/dataflow-types/src/client/controller/storage.rs

```diff
@@ -75,9 +82,6 @@ pub trait StorageController: Debug + Send {
         id: GlobalId,
     ) -> Result<&mut CollectionState<Self::Timestamp>, StorageError>;
 
-    /// Returns the necessary metadata to read a collection
-    fn collection_metadata(&self, id: GlobalId) -> Result<CollectionMetadata, StorageError>;
-
     /// Create the sources described in the individual CreateSourceCommand commands.
     ///
     /// Each command carries the source id, the  source description, an initial `since` read
```

Comment:
doesnt this need to be updated to not mention the `since`?

Reply:
@petrosagg Yep, the since should be removed.